### DISCUSSION TITLE
policy

### DISCUSSION THREAD

**Discussion by xxdyu** [rabbitmq/rabbitmq-server#2444] [Other] [answered]
![1599814303(1)](https://user-images.githubusercontent.com/32918557/92896718-32cb4f00-f44f-11ea-96b4-ebea53f89f6a.jpg)

why?

**Comment by michaelklishin** [comment, marked as answer]
We don't have enough information to suggest much. I cannot reproduce (and I have been working on exchange federation specifically recently). This is not that we have seen reported elsewhere.

My only guess is that the keys are encoded or escaped by an intermediary.
